scenarios2/07_Infinite_Legion.cfg

```diff
@@ -145,7 +145,7 @@
         canrecruit=yes
         side=1
         controller=human
-        recruit=Elvish Fighter,Elvish Archer,Elvish Shaman,Elvish Scout,Elvish Druid,Elvish Sorceress,Elvish Captain,Elvish Hero,Elvish Marksman,Elvish Ranger
+        recruit=Elvish Fighter,Elvish Archer,Elvish Shaman,Elvish Scout,Elvish Lord
```

Comment:
I don't remember there being a possibility to recruit Elvish Lords. Were they even recruitable somewhere?

Reply:
Aha, okay. I didn't remember they were recruitable.